server/tests/ballot_comparison/snapshots/snap_test_cvrs.py

```diff
@@ -181,62 +181,62 @@
     {
         "ballot_position": 2,
         "batch_name": "BATCH1",
-        "imprinted_id": "2-1-2",
+        "imprinted_id": "56_083-212",
```

Comment:
I assume this change is just to match the actual Dominion style?

Reply:
this diff is misleading. the snapshots got reordered, i think because i renamed some tests. let me see if i can fix

Reply:
oh I gotcha. It's not a big deal.

Reply:
ok check now, should be clearer

Reply:
Yep!